slycot/tests/test_ab05md.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose, assert_array_equal, assert_equal
+from scipy import linalg, signal
+
+from slycot import analysis
+
+
+class Test_ab05md:
+
+    @pytest.mark.skip("not implemented yet, TODO")
+    def test_ab05md():
+        pass
```

Comment:
I'm not a fan of these. They pollute the report about skipped tests but increase collection time for no real benefit

Reply:
So this should be deleted again?
Then we have a wrapper, but no test.

Is that OK?

Reply:
I'd rather have no test than a test that does nothing. We already have the missing coverage as indication that there should be one implemented

Reply:
OK